tap2junit/__main__.py

```diff
@@ -74,7 +74,7 @@ def main():
     arg_parser.add_argument(
         "--input",
         "-i",
-        type=argparse.FileType("r"),
+        type=argparse.FileType("r", encoding="UTF-8"),
```

Comment:
```suggestion
        type=argparse.FileType("r", encoding="utf-8"),
```
https://docs.python.org/3/library/codecs.html#standard-encodings

Reply:
I have followed the example here https://docs.python.org/3/library/argparse.html#argparse.FileType, but lower case seems to work as well